Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -1,5 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
+from Configuration.StandardSequences.Reconstruction_cff import *
```

Comment:
Hongxuan Liu has suggested to use directly ESProducer (of those SeverityLevel setings), so 

I presume something like this should be OK ?
 from RecoLocalCalo.HcalRecAlgos.hcalRecAlgoESProd_cfi import *